tokio/src/fs/mocks.rs

```diff
@@ -65,7 +65,7 @@ impl Read for &'_ MockFile {
     }
 }
 
-impl Seek for &'_ MockFile {
+impl Seek for MockFile {
```

Comment:
This seems to cause a compilation failure. Most likely, you can just undo the change.

Reply:
My apologies. That was a mistake. I'm currently working through/experimenting offloading to `spawn_blocking`

Reply:
Okay. Hopefully it isn't too complicated to do that. What you're doing now may be preferable if offloading is very complicated.

Reply:
I will resolve it in just a moment.